corehq/form_processor/models/cases.py

```diff
@@ -96,6 +96,18 @@ def get_case_ids_that_exist(self, domain, case_ids):
                           .values_list('case_id', flat=True))
         return result
 
+    def get_case_ids_in_domain(self, domain, type=None):
```

Comment:
Very nitty: This might be a nice opportunity to rename the `type` param to `case_type`. It is a 1-line method, and we don't call it with `type` as a keyword argument often, so it's OK as it is. It just feels a little uncomfortable.

Reply:
Is your rationale that `type` is a built-in name, and using it masks that name in this function? As a counterpoint, the name of the `CommCareCase` attribute that it maps to is also `type`. I'm inclined to leave it.